### DISCUSSION TITLE
EffectPass documentation

### DISCUSSION THREAD

**Discussion by ixxie** [pmndrs/postprocessing#476] [Q&A] [answered]
The documentation for [ShaderPass](https://pmndrs.github.io/postprocessing/public/docs/class/src/passes/ShaderPass.js~ShaderPass.html) has the following comment:

> If you want to create multiple chained effects, please use EffectPass instead.

I cannot find any mention of an EffectPass in the docs. What am I missing?

**Comment by vanruesc** [comment, marked as answer]
Thanks for pointing that out! There seems to be a problem with ESDoc failing to parse some classes, including the `EffectPass`. Will fix this when I get a chance. Until then, you can take a look at the [source](https://github.com/pmndrs/postprocessing/blob/da28cd5d6800da9e6c10ffb4590a23258d7f766a/src/passes/EffectPass.js#L240) instead.

The comment you're referring to means that it's usually better to write individual [Effects](https://github.com/pmndrs/postprocessing/wiki/Custom-Effects) which can be merged with the `EffectPass` to minimize fullscreen render operation.

**Comment by ixxie** [comment]
Thanks @vanruesc, good to know :)
